csatar-plugins/csatar/models/Scout.php

```diff
@@ -38,40 +40,130 @@ class Scout extends OrganizationBase
+        'foreign_language_knowledge',
+        'occupation',
+        'workplace',
+        'comment',
+        'profile_image',
+        'registration_form',
+        'chronic_illnesses',
+    ];
+
+    protected $nullable = [
+        'user_id',
+        'troop_id',
+        'patrol_id',
+        'name_prefix',
+        'family_name',
+        'given_name',
+        'nickname',
+        'email',
+        'phone',
+        'personal_identification_number',
+        'gender',
+        'is_active',
+        'legal_relationship_id',
+        'special_diet_id',
+        'religion_id',
+        'nationality',
+        'tshirt_size_id',
+        'birthdate',
+        'nameday',
+        'maiden_name',
+        'birthplace',
+        'address_country',
+        'address_zipcode',
+        'address_county',
+        'address_location',
+        'address_street',
+        'address_number',
+        'mothers_name',
+        'mothers_email',
+        'fathers_name',
+        'fathers_email',
+        'legal_representative_name',
+        'legal_representative_email',
+        'elementary_school',
+        'primary_school',
+        'secondary_school',
+        'post_secondary_school',
+        'college',
+        'university',
+        'other_trainings',
+        'foreign_language_knowledge',
+        'occupation',
+        'workplace',
+        'comment',
+    ];
+
     /**
      * @var array Validation rules
      */
     public $rules = [
         'team' => 'required',
-        'family_name' => 'required',
-        'given_name' => 'required',
-        'email' => 'email',
-        'phone' => 'regex:(^[0-9+-.()]{10,}$)',
+        'phone' => 'nullable|regex:(^[0-9+-.()]{10,}$)',
+        'birthdate' => 'required',
         'legal_representative_phone' => 'regex:(^[0-9+-.()]{10,}$)',
         'mothers_phone' => 'regex:(^[0-9+-.()]{10,}$)',
         'fathers_phone' => 'regex:(^[0-9+-.()]{10,}$)',
-        'personal_identification_number' => 'required',
-        'gender' => 'required',
-        'is_active' => 'required',
-        'legal_relationship' => 'required',
-        'religion' => 'required',
-        'birthdate' => 'required',
-        'birthplace' => 'required',
-        'address_country' => 'required',
-        'address_zipcode' => 'required',
-        'address_county' => 'required',
-        'address_location' => 'required',
-        'address_street' => 'required',
-        'address_number' => 'required',
-        'mothers_email' => 'email',
-        'fathers_email' => 'email',
-        'legal_representative_email' => 'email',
+        'mothers_email' => 'email|nullable',
+        'fathers_email' => 'email|nullable',
+        'legal_representative_email' => 'email|nullable',
         'profile_image' => 'image|nullable|max:5120',
         'registration_form' => 'mimes:jpg,png,pdf|nullable|max:1536',
+        'legal_relationship' => 'required',
         'chronic_illnesses' => 'required',
         'special_diet' => 'required',
     ];
```

Comment:
Ezek tovabbra is kotelezoek maradnak?

Reply:
Ezeken mind foreign key van és ahhoz hogy nullable-re tudjuk tenni, a forieign key-t is törölni kellene. Jobbnak láttam nem kivenni.